students/swietczak_monika/lesson_12_debugging/regex_validators.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+def email_validator(some_email):
+    email_regex = re.compile(r"[^@]+@[^@]+\.[^@]+")
+    result = email_regex.match(some_email)
+    if result is None:
+        return False
+    else:
+        return True
+
+
+def phone_number_validator(some_number):
+    phone_regex = re.compile(r"^\d{3}[- ]?\d{3}[- ]?\d{3}$")
+    if phone_regex.match(some_number) is None:
+        return False
+    else:
+        return True
+
+
+def postal_code_validator(some_postal_code):
+    postal_code_regex = re.compile(r"^\d\d-\d\d\d$")
+    if postal_code_regex.match(some_postal_code) is None:
+        return False
+    else:
+        return True
+
+
+def pass_checker(some_password):
+    if re.match(r'^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{8,}$',
```

Comment:
That's an interesting solution! In all the other reviews multiple regular expressions were used. :+1: